static/js/components/CommentAttachmentPreview.jsx

```diff
@@ -272,12 +279,28 @@ const CommentAttachmentPreview = ({
                   theme={darkTheme ? "monokai" : "rjv-default"}
                 />
               )}
-              {supportedType && fileType !== "pdf" && fileType !== "json" && (
-                <FilePreviewerThumbnail
-                  file={{ url: previewUrl }}
-                  hideControls
-                />
-              )}
+              {supportedType &&
+                ["jpeg", "jpg", "png", "gif"].includes(fileType) && (
+                  <img
+                    src={previewUrl}
+                    alt={filename}
+                    style={{
+                      maxHeight: "68vh",
```

Comment:
Where do these values come from? 68vh seems rather specific.

Reply:
I could have a `calc(do_something)` instead, but it's basically what's enough to exclude the app bar, the dialog header, and dialog footer, and the margins between the dialog and the page on top and bottom. It's pretty arbitrary, and again in that PR I want to open that I mentioned in a comment above, I want to have something that figures out the exact height that's available.

Reply:
It's probably a good idea to file issues for items you think you may address in the future. With checklists, if the issue covers multiple items. I'm amazed at how often I intend to fix something and then get distracted along the way.

Reply:
Yes! I agree 100% I used to do it but I should use that again. I'm taking care of deploy right now, but I will have time to open a bunch of issues tomorrow afternoon. I got a batch of feature request that will require a significant number frontend and backend changes, so it will come in handy.